.github/workflows/appengine-deploy.yml

```diff
@@ -52,3 +56,15 @@ jobs:
       - id: test
         name: Test the deloyed URL
         run: curl --fail "${{ steps.deploy.outputs.url }}"
+        
+      - name: Happo Screenshot Testing
+        working-directory: web
+        run: npm run happo-ci-github-actions
+        env:
+          PREVIOUS_SHA: 490c6fa004074f6cf23649f5336f1e9004f4574d
+          CURRENT_SHA: ${{ env.CURRENT_HASH }}
+          CHANGE_URL: ${{ github.event.pull_request.html_url }}
```

Comment:
Same as the commit hash, what if the `github.event` is not a pull_request? Maybe you want to customize a new env variable! :)

Reply:
Changed the url to the commit url, which should cover all the cases of the pull request and the push to main!